frontend/src/metabase/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,60 @@
+import type { ReactNode } from "react";
+import { t } from "ttag";
+import _ from "underscore";
+
+import { Button, Flex, Modal, Text } from "metabase/ui";
+
+interface ConfirmContentProps {
+  opened: boolean | undefined;
+  "data-testid"?: string;
+  title: string | ReactNode;
+  content?: string;
+  message?: string | ReactNode;
+  onClose?: () => void;
+  onConfirm?: () => void;
+  confirmButtonText?: string;
+  confirmButtonPrimary?: boolean;
+}
+
+export const ConfirmationModal = ({
+  opened,
+  "data-testid": dataTestId,
+  title,
+  content,
+  message = t`Are you sure you want to do this?`,
+  onClose = _.noop,
+  onConfirm = _.noop,
+  confirmButtonText = t`Yes`,
+  confirmButtonPrimary = false,
+}: ConfirmContentProps) => (
+  <Modal.Root opened={Boolean(opened)} onClose={onClose}>
+    <Modal.Overlay />
+    <Modal.Content data-testid={dataTestId}>
+      <Modal.Header p="2.5rem 3rem" mb="sm">
```

Comment:
Space between title and content seems a little large. Also title seems a little small?

Old | New
---|---
![Screen Shot 2025-02-25 at 8 27 18 AM](https://github.com/user-attachments/assets/75ff0fbe-d156-4f67-b662-291fa08d85f0) | ![Screen Shot 2025-02-25 at 8 27 24 AM](https://github.com/user-attachments/assets/84179e7e-f459-439c-ac44-c044bc339ca4)

Really, I think we should probably just go with the modal defaults here.